mojo/stdlib/stdlib/algorithm/functional.mojo

```diff
@@ -107,23 +107,25 @@ fn vectorize[
 
     ```mojo
     from algorithm.functional import vectorize
+    from memory import UnsafePointer
 
     # The amount of elements to loop through
     alias size = 10
     # How many Dtype.int32 elements fit into the SIMD register (4 on 128bit)
-    alias simd_width = simdwidthof[DType.int32]()
+    alias simd_width = 4
```

Comment:
From feedback internally:
> can we keep this as `simdwidthof` instead of a magic number. 128 bits is not the simd width of avx512 for example

Reply:
I agree. I'd leave it as you have it; `simdwidth[DType.int32]()` with a comment.